### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Alek99**
Make the guide more concise and point to notion.

### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/pynecone-io/pynecone/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/pynecone-io/pynecone/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [ ] Does your submission pass the tests? 
- [ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,75 +1,47 @@
-# Welcome to Reflex contributing guide! 🥳
+# Reflex Contributing Guidelines
 
-## Getting started
+For an extensive guide on the different ways to contribute to Reflex please visite our notion's [Contributing Guide](https://www.notion.so/reflex-dev/2107ab2bc166497db951b8d742748284?v=f0eaff78fa984b5ab15d204af58907d7).
```
Comment (picklelo) [inline CONTRIBUTING.md:3]: typo: "visit our Notion's". I think we can reword this also to "see our Contributing Guide on Notion"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,75 +1,47 @@
...
-### Issues
-
-* #### Create a new issue
-
-    If you spot a problem with anything in Reflex feel free to create an issue. Even if you are not sure if its a problem with the framework or your own code, create an issue and we will do our best to answer or resolve it.
-
-* #### Solve an issue
-
-    Scan through our [existing issues](https://github.com/pynecone-io/pynecone/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix. Any large issue changing the compiler of Reflex should brought to the Reflex maintainers for approval
+## Runing a Local Build of Reflex
```
Comment (picklelo) [inline CONTRIBUTING.md:7]: Running

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,75 +1,47 @@
...
-    If you spot a problem with anything in Reflex feel free to create an issue. Even if you are not sure if its a problem with the framework or your own code, create an issue and we will do our best to answer or resolve it.
-
-* #### Solve an issue
-
-    Scan through our [existing issues](https://github.com/pynecone-io/pynecone/issues) to find one that interests you. You can narrow down the search using `labels` as filters. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix. Any large issue changing the compiler of Reflex should brought to the Reflex maintainers for approval
+## Runing a Local Build of Reflex 
+Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
 
-Thank you for supporting Reflex!🎊
+**Pre-requisites:**
```
Comment (picklelo) [inline CONTRIBUTING.md:10]: nit: don't need hyphen, also don't think we need the colon at the end

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,75 +1,47 @@
...
+- Python >= 3.7
+- Poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
```
Comment (picklelo) [inline CONTRIBUTING.md:12]: Technically you shouldn't need poetry to contribute. Should we mention that it's optional but that this guide will show the poetry way?

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,75 +1,47 @@
...
-
-Install your local Reflex build:
+**2. Install your local Reflex build:**
 ``` bash
 poetry install
 ```
-
-Now create an examples folder so you can test the local Python build in this repository:
+**3. Now create an examples folder so you can test the local Python build in this repository.**
+* We have the examples folder in the .gitignore, so your changes in reflex/examples won't be reflected in your commit.
```
Comment (picklelo) [inline CONTRIBUTING.md:26]: Use backticks for `examples`, `.gitignore`, and `reflex/examples`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,75 +1,47 @@
...
-Thank you for supporting Reflex!🎊
+**Pre-requisites:**
+- Python >= 3.7
+- Poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
 
-## 💻 How to Run a Local Build of Reflex 
-Here is a quick guide to how the run Reflex repo locally so you can start contributing to the project.
 
-First clone Reflex:
+**1.  Clone Reflex and navigate into the repo:**
```
Comment (picklelo) [inline CONTRIBUTING.md:15]: Extra space after the 1.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -1,75 +1,47 @@
...
-Install poetry version >= 1.4.0 and add it to your path (see [Poetry Docs](https://python-poetry.org/docs/#installation) for more info).
-
-Install your local Reflex build:
+**2. Install your local Reflex build:**
 ``` bash
 poetry install
 ```
-
-Now create an examples folder so you can test the local Python build in this repository:
+**3. Now create an examples folder so you can test the local Python build in this repository.**
```
Comment (picklelo) [inline CONTRIBUTING.md:25]: I feel we can make this line more concise

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -79,28 +51,14 @@ What to unit test?
 
 Once you solve a current issue or improvement to Reflex, you can make a pr, and we will review the changes. 
 
-Before submitting, a pull request, ensure the following steps are taken and test passing.
+Before submitting, a pull request, ensure the following steps are taken and test passing. To do this install pre-commit and run the tests.
```
Comment (picklelo) [inline CONTRIBUTING.md:54]: Use backticks for `pre-commit`

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -79,28 +51,14 @@ What to unit test?
...
 ```
 
-Consider installing git pre-commit hooks so Ruff, Pyright, Darglint and Black will run automatically before each commit.
-Note that pre-commit will only be installed when you use a Python version >= 3.8.
 ``` bash
-pre-commit install
+poetry run pre-commit
 ```
 
 That's it you can now submit your pr. Thanks for contributing to Reflex!
```
Comment (picklelo) [inline CONTRIBUTING.md:64]: we should capitalize PR here

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -1,75 +1,47 @@
...
 poetry run reflex init
 poetry run reflex run
 ```
-
 All the changes you make to the repository will be reflected in your running app.
-* We have the examples folder in the .gitignore, so your changes in reflex/examples won't be reflected in your commit.
 
-## 🧪 Testing and QA
 
-Any feature or significant change added should be accompanied with unit tests.
```
Comment (picklelo) [inline CONTRIBUTING.md:68]: Why did we remove this? Is it because it's repeated below?

**Review by picklelo — COMMENTED** [review]
Nice update, mostly formatting comments

File: CONTRIBUTING.md (hunk 10 above)
Comment (Alek99) [reply CONTRIBUTING.md:68]: Yes

**Review by picklelo — APPROVED** [review]

**Pull request merged**